core/sds-aspect-model-document-generators/src/main/java/io/openmanufacturing/sds/aspectmodel/generator/AspectModelHelper.java

```diff
@@ -0,0 +1,180 @@
+/*
```

Comment:
All Java files do not seem to follow the code style used in the project. Please adjust.

Reply:
What do you mean. I changed only the name and added an additonal method?

Reply:
done